dbt/task/compile.py

```diff
@@ -6,16 +6,44 @@
 import dbt.project
 from collections import defaultdict
 
+from functools import partial
+import networkx as nx
+
 CREATE_STATEMENT_TEMPLATE = """
 create {table_or_view} {schema}.{identifier} {dist_qualifier} {sort_qualifier} as (
     {query}
 );"""
 
+class Linker(object):
```

Comment:
agreed -- it's a thin wrapper around the nx digraph. Not convinced that it's hurting anyone as is right now though :)